tests/server/flow-event.js

```diff
@@ -430,48 +428,49 @@ define([
     },
 
     'call flowEvent without optional flow data': {
-      beforeEach () {
+      beforeEach() {
```

Comment:
In my usual grumpy manner, I'm going to point out that this massive diff would be easier to review if it didn't also contain a bunch of unrelated formatting changes like removing the space on all these `beforeEach` methods. No biggie obviously, just more to scroll through.